### ISSUE TITLE
Support sha384

### ISSUE THREAD

**Issue by chucklu** [gurnec/HashCheck#72]
The sri hash site https://www.srihash.org/ generate sha384 by default, and https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity also use sha384 as example.
It would be great if this tool can support sha384.

**Comment by LuxLOL** [comment]
It's really sad that no one keeps this nice tool up-to-date. This is the best Hash Check tool out their. Because you can very easy compare/save multiple hashes at once and sort them as you wish.

**Comment by sergeevabc** [comment]
@chucklu, srihash.org has three hashes to choose from: sha256 and sha512 besides sha384, the former ones are present in HashCheck and there is no problem with them. Don't let paranoids infect your brain with 'more bits of security' stuff. But more important is that you have to encode sha384 output with base64 as well to use later inside a specific html tag. HashCheck is far from dealing with tricky network schemes, it's more grounded in processing local files and verifying its checksums.

But the way, are you still using this app, or moved elsewhere (e.g. [OpenHashTab](https://github.com/namazso/OpenHashTab))?

**Comment by chucklu** [comment]
@sergeevabc yes, I know that srihash.org site provided three hashes, but as I said sha384 is the default one. The other two needs to manually switch.

I know this tool is used to verify the sha of local files, and we check the .js file for the Subresource Integrity. we also use it to verify the file integrity when we need to compare the files shared.

I am still using this app, as it's integrated with shell. It's the first time I heard OpenHashTab, do you recommend it?

**Comment by sergeevabc** [comment]
> It's the first time I heard OpenHashTab, do you recommend it?

You should definitely try it. It can be safely installed and removed later if necessary.
Having said that, I don't mean that you'll like it more, but at least it's evolving, although I'm not thrilled with its design. Unfortunately, none of the hashing calculators for Windows with a graphical interface have satisfied me so far.
